tests/dashbio_demos/app_speck.py

```diff
@@ -1,4 +1,5 @@
 import os
+import base64
```

Comment:
You might as well skip a line after this import ;)

Reply:
Done in 99551cd!

Reply:
'space' --> 'blank line' but ok :)